ci-operator/step-registry/gcp/provision/bastionhost/gcp-provision-bastionhost-commands.sh

```diff
@@ -74,7 +74,8 @@ MACHINE_TYPE="n2-standard-2"
 echo "${REGION}" >> "${SHARED_DIR}/proxyregion"
 
 bastion_name="${CLUSTER_NAME}-bastion"
-gcloud compute instances create "${bastion_name}" \
+CMD="gcloud compute instances create ${bastion_name} \
+  --hostname=${bastion_name}.test.com \
```

Comment:
Is this a MUST? If not set it, what would happen?

Reply:
Without it, the bastion host would have "localhost" as its hostname, so I'd like to keep it. 
```
[core@localhost ~]$ hostname
localhost
[core@localhost ~]$

Reply:
@jianli-wei Does `localhost` bring some trouble? Because the bastion would also run proxy container, I am not sure if a full-domain hostname for container host bring some side effect for dns resolving stuff when the bastion is taking a role of proxy, personally I prefer to keep `localhost` to be safe.